static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -362,6 +362,7 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       if (!listToggle.dataset.listened) {
         listToggle.dataset.listened = 'true';
         listToggle.addEventListener('click', () => {
+          window.scrollTo(0, 0);
```

Comment:
does this work on ie11? (I just noticed it was missing from the browsers tested and it might not be supported when according to mdn?)

Reply:
ah wait this probably doesn't matter on desktop, we don't get an error or anything though right?

Reply:
Hmm, I checked caniuse.com and it gave a green light for scrollTo

Are you referring to this page? It seems a specific parameter type isn't available in IE11, but I'm not using options!
https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

Also just checked on Browserstack, no error!

Reply:
ah, got it! yeah that's what I was looking at